docs/docs/Integrations/integrations-mcp.md

```diff
@@ -49,28 +91,123 @@ This code adds a new MCP server called `langflow` and starts the [mcp-sse-shim](
+```text
+I'll help you explore a resume using the Document QA for Resume flow, which is specifically designed for analyzing resumes.
+Let me call this tool.
+```
+6. Click **Run tool** to continue. The agent requests permissions when necessary.
+```
+Based on the resume, here's a comprehensive breakdown of the experience:
+```
+7. Ask about a different resume.
+You've provided enough information in the description for the agent to make the correct decision:
+```text
+I notice you're asking about Emily's job experience.
+Based on the available tools, I can see there is a Document QA for Resume flow that's designed for analyzing resumes.
+However, the description mentions it's for "Alex's resume" not Emily's. I don't have access to Emily's resume or job experience information.
+```
+
 ## Connect an Astra DB MCP server to Langflow
 
-Use the [MCP stdio component](/components-tools#mcp-tools-stdio) to connect Langflow to a [Datastax Astra DB MCP server](https://github.com/datastax/astra-db-mcp).
+Use the [MCP server component](/components-tools#mcp-server) to connect Langflow to a [Datastax Astra DB MCP server](https://github.com/datastax/astra-db-mcp).
 
 1. Install an LTS release of [Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
 2. Create an [OpenAI](https://platform.openai.com/) API key.
 3. Create an [Astra DB Serverless (Vector) database](https://docs.datastax.com/en/astra-db-serverless/databases/create-database.html#create-vector-database), if you don't already have one.
 4. Get your database's **Astra DB API endpoint** and an **Astra DB application token** with the Database Administrator role. For more information, see [Generate an application token for a database](https://docs.datastax.com/en/astra-db-serverless/administration/manage-application-tokens.html#database-token).
-5. Add your **Astra DB application token** and **Astra API endpoint** to Langflow as [global variables](/configuration-global-variables).
-6. Create a [Simple agent starter project](/starter-projects-simple-agent).
-7. Remove the **URL** tool and replace it with an [MCP stdio component](/components-tools#mcp-tools-stdio) component.
+5. Create a [Simple agent starter project](/starter-projects-simple-agent).
+6. Remove the **URL** tool and replace it with an [MCP server component](/components-tools#mcp-server) component.
 The flow should look like this:
-![MCP stdio component](/img/mcp-stdio-component.png)
-8. In the **MCP stdio** component, in the **MCP command** field, add the following code:
+![MCP stdio component](/img/mcp-server-component.png)
+7. In the **MCP server** component, in the **MCP command** field, add the following code.
+Replace the values for `ASTRA_TOKEN` and `ASTRA_ENDPOINT` with the values from your Astra database.
 
 ```plain
-npx -y @datastax/astra-db-mcp
+env ASTRA_DB_APPLICATION_TOKEN=ASTRA_TOKEN ASTRA_DB_API_ENDPOINT=ASTRA_ENDPOINT npx -y @datastax/astra-db-mcpnpx -y @datastax/astra-db-mcp
```

Comment:
Feels like this astra-db-mxpnpx thing should be its own thing somewhere else in the docs. Mixing it up with the native langflow functionality is confusing.

Reply:
Good call. I moved it to its own page